graphql/resolver.go

```diff
@@ -179,11 +179,12 @@ func New(apiURL string) Config {
 	c.Directives.RequireProjectFieldAccess = func(ctx context.Context, obj interface{}, next graphql.Resolver) (res interface{}, err error) {
```

Comment:
should this directive be renamed to like RequireProjectViewAccess or etc, since we aren't calling it by field anymore?